omero/developers/Java.rst

```diff
@@ -235,6 +235,33 @@ Given a plate ID, load the wells.
         //Do something
     }
 
+-  **Retrieve Annotations.**
+
+Load the MapAnnotations (Key Value pairs) for a the logged in user
+
+::
+
+    BrowseFacility browse = gateway.getFacility(BrowseFacility.class);
```

Comment:
There should be a magical way to include code snippets directly from our source repo? (Copy-and-paste is a maintenance issue.)

Reply:
Oh my goodness, the page is full of it, isn't it? Outwith this PR then, noted on https://trello.com/c/JnGl5lPx/630-eliminate-copied-code#comment-5b164fd2e75687f8c8452862.